teachertool/src/services/backendRequests.ts

```diff
@@ -2,6 +2,21 @@ import { stateAndDispatch } from "../state";
 import { ErrorCode } from "../types/errorCode";
 import { logError } from "./loggingService";
 
+export async function fetchJsonDocAsync<T = any>(url: string): Promise<T | undefined> {
+    try {
+        // TODO: Prepend CDN origin if not localhost
+        const response = await fetch(url);
```

Comment:
I'm wondering if we should update loadTestableCollectionFromDocsAsync to call into this as well on line 74. Can be done in a separate change, if preferred.